packages/core/admin/admin/src/pages/SettingsPage/pages/ApplicationInfosPage/components/Form/tests/index.test.js

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import { IntlProvider } from 'react-intl';
+import { render as renderTL } from '@testing-library/react';
+import { ThemeProvider, lightTheme } from '@strapi/design-system';
+
+import Form from '..';
+
+const PROJECT_SETTINGS_DATA_FIXTURES = {
+  authLogo: {
+    ext: '.jpeg',
+    height: 250,
+    name: 'authLogo.jpeg',
+    size: 46.26,
+    url: 'uploads/auth.jpeg',
+    width: 340,
+  },
+  menuLogo: {
+    ext: '.jpeg',
+    height: 250,
+    name: 'menuLogo.jpeg',
+    size: 46.26,
+    url: 'uploads/menu.jpeg',
+    width: 340,
+  },
+};
+
+jest.mock('@strapi/helper-plugin', () => ({
+  useTracking: jest.fn(() => ({ trackUsage: jest.fn() })),
+}));
+jest.mock('../../../../../../../hooks', () => ({
+  useConfigurations: jest.fn(() => ({
+    logos: {
+      menu: { custom: 'customMenuLogo.png', default: 'defaultMenuLogo.png' },
+      auth: { custom: 'customAuthLogo.png', default: 'defaultAuthLogo.png' },
+    },
+  })),
+}));
+
+const render = (props) =>
+  renderTL(
+    <ThemeProvider theme={lightTheme}>
+      <IntlProvider locale="en" messages={{}} textComponent="span">
+        <Form canUpdate projectSettingsStored={PROJECT_SETTINGS_DATA_FIXTURES} {...props} />
+      </IntlProvider>
+    </ThemeProvider>
+  );
+
+describe('ApplicationInfosPage | Form', () => {
+  it('should render and match snapshot', () => {
+    const { container } = render();
+    expect(container).toMatchSnapshot();
+  });
```

Comment:
As this page is quite big, I think we don't have to take a snapshot test here and instead do it more granularly as you do below.

Reply:
works for me :)